medical/security/medical_security.xml

```diff
@@ -7,40 +7,38 @@
     </record>
 
     <record id="group_medical_user" model="res.groups">
-        <field name="name">Medical User</field>
```

Comment:
I'm not sure what you mean. This is one of the lowest of the security groups, it should be near the top.